content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,124 @@
+
+Dual-stack support in Kubernetes means pods, services, and nodes can get IPv4 addresses and IPv6 addresses. In Kubernetes 1.21 dual-stack support is graduating from [alpha](/docs/concepts/services-networking/dual-stack/) to beta and is now enabled by default.
+
+### Graceful Node Shutdown
+[Graceful Node shutdown](/docs/concepts/architecture/nodes/#graceful-node-shutdown) is graduating to beta with this release (and will now be available to a much larger group of users!). This is a hugely beneficial feature that allows Kubelet to be aware of Node shutdown, and gracefully terminate pods that are scheduled to that Node.
+
+Currently, when a node shuts down, pods do not follow the expected termination lifecycle and are not shutdown gracefully. This can introduce problems with a lot of different workloads. Going forward, Kubelet will be able to propagate the shutdown signal to pods ensuring they can terminate as gracefully as possible.
+
+### PersistentVolume Health Monitor
+Physical Volumes (PV) are commonly used in applications to get local, file-based storage. They can be used in many different ways and help users migrate applications without needing to re-write storage backends.
+
+Kubernetes 1.21 has a new alpha feature which allows PVs to be monitored for health of the volume and marked accordingly if the volume becomes unhealthy. Workloads will be able to react to the health state to protect data from being written or read from a volume that is unhealthy.
+
+### Reducing Kubernetes Build Maintenance
+Previously Kubernetes has maintained multiple build systems. This has often been a source of friction and complexity for new and current contributors.
+
+Over the last release cycle, a lot of work has been put in to simplify the build process, and standardize on the native Golang build tools. This should empower broader commmunity maintenance, and lower the barier to entry for new contributors.
+
+## Major Changes
+
+### PodSecurityPolicy Deprecation
+In Kubernetes 1.21, PodSecurityPolicy is deprecated. As with all Kubernetes feature deprecations, PodSecurityPolicy will continue to be available and fully-functional for several more releases. PodSecurityPolicy, previously in the beta stage, is planned for removal in Kubernetes 1.25.
+
+What's next? We're developing a new feature, tentatively called PodIsolationPolicy, that covers the key PodSecurityPolicy use cases with greatly improved ergonomics and maintainability. To learn more, check out the PSP feature blog [here]().
+
+### TopologyKeys Deprecation
+The service field topologyKeys is being deprecated and replaced with a new feature called topology-aware hints, which will be released as alpha in Kubernetes 1.21. You can read more details about the proposed changes in the [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/2433-topology-aware-hints/README.md)
```

Comment:
How about:
```suggestion
The Service field `topologyKeys` is now deprecated; all the component features that used this field were previously alpha, and are now also deprecated.

We've replaced `topologyKeys` with a way to implement topology-aware routing, called topology-aware hints.  Topology-aware hints are an alpha featuee in Kubernetes 1.21. You can read more details about the replacement feature in [Topology Aware Hints](/docs/concepts/services-networking/topology-aware-hints/); the related [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/2433-topology-aware-hints/README.md) explains the context for why we switched.
```

Reply:
@salaxander : I have made changes to the blurb and committed it, please can you cross-check with the Enhancement owner once?

Reply:
@divya-mohan0209 This looks good